sorting/sorting.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("The program sorts three numbers.")
+	fmt.Print("Enter the numbers: ")
+	var a, b, c float64
+	fmt.Scan(&a, &b, &c)
+	if a < b && a < c {
```

Comment:
What if we had 5 or even 10 numbers here? Can you try to come up with a solution that scales kinda better (still it will be lots of repeated code, but I think a bit less).

_Hint: You can swap variable values._